pkg/runtime/wfengine/workflow.go

```diff
@@ -391,10 +399,12 @@ func (wf *workflowActor) runWorkflow(ctx context.Context, actorID string, remind
 			return newRecoverableError(errExecutionAborted)
 		}
 	}
+	wfLogger.Debugf("workflow actor '%s': workflow execution returned with status %s instanceId '%s'", actorID, runtimeState.RuntimeStatus().String(), wi.InstanceID)
 
 	// Increment the generation counter if the workflow used continue-as-new. Subsequent actions below
 	// will use this updated generation value for their duplication execution handling.
 	if runtimeState.ContinuedAsNew() {
+		wfLogger.Debugf("workflow actor '%s': workflow with instance Id '%s' continued as new", actorID, wi.InstanceID)
```

Comment:
instanceID (with no space) has been used throughout the logs, but instance ID is used here. I'm okay with either but lets stick with one

Reply:
Done